### PR TITLE
Re-enable tests for legacy/non-Angular JavaScript

### PR DESCRIPTION
**Pull request by myabc**
These tests were written in QUnit, but had broken dependences and were not executed by our Karma runner.

### CODE REVIEW

**Comment by marutosi** [comment]
> >    Stupid.
> >    Should not be in description.
> >    All deails should be in ticket and revisions.
> 
> even better! where are your tickets? i've never seen one, have a link?
> anyway, just pasting the summary of your pull request in the description field doesn't hurt, right?!

#1362
Where is ticket link?

**Comment by myabc** [comment]
@marutosi please feel free to review this PR anyway though!

File: karma.conf.js [hunk 1]
```diff
@@ -94,16 +95,25 @@ module.exports = function(config) {
...
 
+      'app/assets/javascripts/autocompleter.js',
+      'app/assets/javascripts/members_select_boxes.js',
+      'app/assets/javascripts/openproject.js',
+      'app/assets/javascripts/timelines_select_boxes.js',
 
       'app/assets/javascripts/date-en-US.js',
 
+      //'vendor/assets/components/jquery-mockjax/jquery.mockjax.js',
+
```
Comment (hschink) [inline karma.conf.js]: I guess we can remove this comment

**Comment by hschink** [comment]
Good job resurrecting these tests! :+1:

File: karma.conf.js (hunk 1 above)
Comment (myabc) [inline karma.conf.js]: @hschink good point. It's gone now (3956ee7).

**Pull request merged**